config/default.yml

```diff
@@ -23,6 +23,9 @@ RSpec/ExampleWording:
     not: does not
   IgnoredWords: []
 
+RSpec/BeforeAfterAll:
+  Description: 'Avoid the use of before/after(:all) as they are not rolled back and may lead to database state leaking between examples.'
```

Comment:
I don't think this should talk about the database. rubocop-rspec is for any ruby code. Maybe talk about global state and how test state should be set up and torn down before and after each test?